tests/acceptance/blocks/filteroptionscomponent.js

```diff
@@ -0,0 +1,75 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * This class models user interactions with the {@link FilterOptionsComponent}.
+ */
+export default class FilterOptionsComponentBlock {
+  constructor (container) {
+    this._container = container || '';
+  }
+
+  /**
+   * Returns whether this FilterOptions is expanded or not.
+   */
+  async isExpanded () {
+    return this._Selector('.yxt-FilterOptions--collapsed').exists;
+  }
+
+  /**
+   * Return a selector within this block's container context.
+   * @param {string} selector selector for desired element
+   */
+  _Selector (selector) {
+    return Selector(`${this._container} ${selector}`);
```

Comment:
You can also create a private var this._selector = Selector(container) in the constructor and then do this._selector.find(selector) here. That will scope it too.